packages/data/walkthroughs/9.9.9.json

```diff
@@ -33,27 +35,11 @@
     },
     "9992": {
       "sectionTitle": "9.9.9.2 Two Separate Exits",
-      "sectionQuestions": [
-        "P11",
-        "P12",
-        "P13",
-        "P14",
-        "P15",
-        "P16",
-        "P17"
-      ]
+      "sectionQuestions": ["P11", "P12", "P13", "P14", "P15", "P16", "P17"]
     },
     "9993": {
       "sectionTitle": "9.9.9.3 Shared Egress Facilities",
-      "sectionQuestions": [
-        "P18",
-        "P19",
-        "P20",
-        "P21",
-        "P22",
-        "P23",
-        "P24"
-      ]
+      "sectionQuestions": ["P18", "P19", "P20", "P21", "P22", "P23", "P24"]
```

Comment:
Did vscode auto do this for you or was this manual. My IDE formats it one on each line.

Reply:
Yes mine did that automatically when I saved.

Reply:
Alright. I updated my IDE to "wrap if long" hopefully it'll leave it as is from now on.